src/frontends/paddle/tests/test_models/gen_scripts/generate_reverse.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# reverse paddle model generator
+#
+import numpy as np
+from save_model import saveModel
+import sys
+import paddle
+
+def reverse_static(name: str, x, axis, dtype="float32"):
+    paddle.enable_static()
+
+    with paddle.static.program_guard(paddle.static.Program(), paddle.static.Program()):
+        node_x = paddle.static.data(name='x', shape=x.shape, dtype=dtype)
+        out = paddle.fluid.layers.reverse(node_x, axis)
+
+        cpu = paddle.static.cpu_places(1)
+        exe = paddle.static.Executor(cpu[0])
+        # startup program will call initializer to initialize the parameters.
+        exe.run(paddle.static.default_startup_program())
+
+        outs = exe.run(
+            feed={'x': x},
+            fetch_list=[out])
+
+        saveModel(name, exe, feedkeys=['x'], fetchlist=[out], inputs=[x], outputs=[outs[0]], target_dir=sys.argv[1])
+
+    return outs[0]
+
+def reverse_dynamic(name, axis, dtype="float32"):
```

Comment:
Have you checked the generated paddle model that they are of dynamic input shapes?
You could refer to nms, slice ops. It is better to keep only one test procedure to get both static and dynamic model.

Reply:
- Yes. Have checked and it is dynamic shape.

- Okay. Have integrated the test cases.